recipes/kliff/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "kliff" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5df8088517cd45f6bb893ced3a483a84ccb33f89815f6e7f3b55c0478289b91f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} 
+  host:
+    - python
+    - pip
+    - pybind11
+  run:
+    - python
+    - scipy
+    - pybind11
+    - pytest
+    - kim-api
```

Comment:
https://github.com/mjwen/kliff/blob/v0.1.4/docs/source/installation.rst mentions `kimpy`. Is this something we need here?

Reply:
Yes, it should be `kimpy` rather than the `kim-api` package.